src/generated/event-payloads.ts

```diff
@@ -862,10 +862,6 @@ export declare module EventPayloads {
     open_issues: number;
     watchers: number;
     default_branch: string;
-    allow_squash_merge?: boolean;
-    allow_merge_commit?: boolean;
-    allow_rebase_merge?: boolean;
-    delete_branch_on_merge?: boolean;
```

Comment:
I think these have been added recently, I think `@octokit/webhook-definitions` might not have been up-to-date when you run it?

Reply:
Solved on [`c2b71c3` (#113)](https://github.com/octokit/webhooks.js/pull/113/commits/c2b71c3bbbc99ad5aa512248cad14681eca103ce)